target/linux/realtek/files-5.10/arch/mips/rtl838x/setup.c

```diff
@@ -13,6 +13,7 @@
 #include <linux/init.h>
 #include <linux/clkdev.h>
 #include <linux/clk-provider.h>
+#include <linux/clk.h>
```

Comment:
nit; i think you'd want to sort this above clkdev.h to keep alphabetical sorting (which the rest of this file doesn't follow anyway :p

Reply:
sorted

Reply:
i think your IDE sorts incorrectly? clk.h should be above clk-dev :p e.g.
```
#include <linux/clk-provider.h>
#include <linux/clk.h>
#include <linux/clkdev.h>
```
is actually what vim says it should be